### PR TITLE
[ko] Update outdated files in dev-1.23-ko.1 (M43-M54)

### PR DESCRIPTION
**Pull request by jihoon-seo**
- Related issue: #30845 (**dev-1.23-ko.1**)

> * [ ]  M43.  `content/en/docs/contribute/advanced.md`  | 5(+XS) 5(-)
> * [ ]  M44.  `content/en/docs/contribute/new-content/_index.md`  | 3(+XS) 1(-)
> * [ ]  M45.  `content/en/docs/contribute/participate/pr-wranglers.md`  | 8(+XS) 1(-)
> * [ ]  M46.  `content/en/docs/contribute/review/for-approvers.md`  | 1(+XS) 1(-)
> * [ ]  M47.  `content/en/docs/reference/_index.md`  | 2(+XS) 2(-)
> * [ ]  M48.  `content/en/docs/reference/access-authn-authz/authorization.md`  | 1(+XS) 1(-)
> * [ ]  M49.  `content/en/docs/reference/command-line-tools-reference/feature-gates.md`  | 73(+M) 23(-)
> * [ ]  M50.  `content/en/docs/reference/command-line-tools-reference/kube-proxy.md`  | 11(+S) 95(-)
> * [ ]  M51.  `content/en/docs/reference/glossary/pod-disruption.md`  | 7(+XS) 2(-)
> * [ ]  M52.  `content/en/docs/reference/glossary/quantity.md`  | 3(+XS) 2(-)
> * [ ]  M53.  `content/en/docs/reference/glossary/secret.md`  | 1(+XS) 1(-)
> * [ ]  M54.  `content/en/docs/reference/kubectl/cheatsheet.md`  | 25(+S) 7(-)

---

- Related PR: #30852 (**dev-1.22-ko.4**)

/language ko

### CODE REVIEW

File: content/ko/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -769,6 +804,8 @@ kubelet과 같은 컴포넌트의 기능 게이트를 설정하려면, 기능 
 - `HyperVContainer`: 윈도우 컨테이너를 위한
   [Hyper-V 격리](https://docs.microsoft.com/ko-kr/virtualization/windowscontainers/manage-containers/hyperv-container)
   기능을 활성화한다.
+- `IdentifyPodOS`: 파드 OS 필드를 지정할 수 있게 한다. 이를 통해 API 서버 관리 시 파드의 OS를 정석적인 방법으로 알 수 있다. 
+  쿠버네티스 {{< skew currentVersion >}}에서, `pod.spec.os.name`에 사용할 수 있는 값은 `windows`와 `linux`이다.
```
Comment (yoonian) [inline content/ko/docs/reference/command-line-tools-reference/feature-gates.md:808]:
```suggestion
  쿠버네티스 {{< skew currentVersion >}}에서, `pod.spec.os.name` 에 사용할 수 있는 값은 `windows` 와 `linux` 이다.
```

File: content/ko/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -758,6 +792,7 @@ kubelet과 같은 컴포넌트의 기능 게이트를 설정하려면, 기능 
   파드를 정상적으로 종료하려고 시도한다. 자세한 내용은
   [Graceful Node Shutdown](/ko/docs/concepts/architecture/nodes/#그레이스풀-graceful-노드-셧다운)을
   참조한다.
+- `GRPCContainerProbe`: 활성 프로브(Liveness Probe), 준비성 프로브(Readiness Probe), 스타트업 프로브(Startup Probe)에 대해 gRPC 프로브를 활성화한다. [활성/준비성/스타트업 프로브 구성하기](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe)를 참조한다.
 - `HPAContainerMetrics`: `HorizontalPodAutoscaler`를 활성화하여 대상 파드의
```
Comment (yoonian) [inline content/ko/docs/reference/command-line-tools-reference/feature-gates.md:796]:
```suggestion
- `HPAContainerMetrics`: `HorizontalPodAutoscaler` 를 활성화하여 대상 파드의
```

File: content/ko/docs/reference/glossary/secret.md [hunk 3]
```diff
@@ -15,4 +15,4 @@ tags:
 
 <!--more--> 
 
-민감한 정보를 사용하는 방식에 대해 더 세밀하게 제어할 수 있으며, 유휴 상태의 [암호화](/docs/tasks/administer-cluster/encrypt-data/#ensure-all-secrets-are-encrypted)를 포함하여 우발적인 노출 위험을 줄인다. {{< glossary_tooltip text="파드(Pod)" term_id="pod" >}}는 시크릿을 마운트된 볼륨의 파일로 참조하거나, 파드의 이미지를 풀링하는 kubelet이 시크릿을 참조한다. 시크릿은 기밀 데이터에 적합하고 [컨피그맵](/docs/tasks/configure-pod-container/configure-pod-configmap/)은 기밀이 아닌 데이터에 적합하다.
+민감한 정보를 사용하는 방식에 대해 더 세밀하게 제어할 수 있으며, 우발적인 노출 위험을 줄인다. 시크릿 값은 기본적으로 base64 문자열로 인코딩되어 암호화되지 않은 채로 저장되지만, [안전하게 암호화](/docs/tasks/administer-cluster/encrypt-data/#ensure-all-secrets-are-encrypted)되도록 설정할 수 있다. {{< glossary_tooltip text="파드" term_id="pod" >}}는 볼륨 마운트 내의 파일 형태로 시크릿에 접근하며, 시크릿은 또한 kubelet이 파드를 위해 이미지를 풀링할 때에도 사용될 수 있다. 시크릿은 기밀 데이터를 다루는 용도로 적합으며, [컨피그맵](/docs/tasks/configure-pod-container/configure-pod-configmap/)은 기밀이 아닌 데이터를 다루는 용도로 적합하다.
```
Comment (yoonian) [inline content/ko/docs/reference/glossary/secret.md:18]:
```suggestion
민감한 정보를 사용하는 방식에 대해 더 세밀하게 제어할 수 있으며, 우발적인 노출 위험을 줄인다. 시크릿 값은 기본적으로 base64 문자열로 인코딩되어 암호화되지 않은 채로 저장되지만, [안전하게 암호화](/docs/tasks/administer-cluster/encrypt-data/#ensure-all-secrets-are-encrypted)되도록 설정할 수 있다. {{< glossary_tooltip text="파드" term_id="pod" >}}는 볼륨 마운트 내의 파일 형태로 시크릿에 접근하며, 시크릿은 또한 kubelet이 파드를 위해 이미지를 풀링할 때에도 사용될 수 있다. 시크릿은 기밀 데이터를 다루는 용도로 적합하며, [컨피그맵](/docs/tasks/configure-pod-container/configure-pod-configmap/)은 기밀이 아닌 데이터를 다루는 용도로 적합하다.
```

File: content/ko/docs/reference/kubectl/cheatsheet.md [hunk 4]
```diff
@@ -320,6 +323,24 @@ kubectl exec my-pod -c my-container -- ls /         # 기존 파드에서 명령
...
+
+```bash
+kubectl cp /tmp/foo_dir my-pod:/tmp/bar_dir            # 로컬 디렉토리 /tmp/foo_dir 를 현재 네임스페이스의 my-pod 파드 안의 /tmp/bar_dir 로 복사
+kubectl cp /tmp/foo my-pod:/tmp/bar -c my-container    # 로컬 파일 /tmp/foo 를 my-pod 파드의 my-container 컨테이너 안의 /tmp/bar 로 복사
+kubectl cp /tmp/foo my-namespace/my-pod:/tmp/bar       # 로컬 파일 /tmp/foo 를 my-namespace 네임스페이스의 my-pod 파드 안의 /tmp/bar 로 복사
+kubectl cp my-namespace/my-pod:/tmp/foo /tmp/bar       # my-namespace 네임스페이스의 my-pod 파드 안의 파일 /tmp/foo 를 로컬의 /tmp/bar 로 복사
+```
+{{< note >}}
+`kubectl cp` 명령을 사용하려면 컨테이너 이미지에 'tar' 바이너리가 포함되어 있어야 한다. 'tar'가 없으면, `kubectl cp`는 실패할 것이다.
+심볼릭 링크, 와일드카드 확장, 파일 모드 보존과 같은 고급 사용 사례에 대해서는 `kubectl exec` 를 고려해볼 수 있다.
```
Comment (yoonian) [inline content/ko/docs/reference/kubectl/cheatsheet.md:336]:
```suggestion
심볼릭 링크, 와일드카드 확장, 파일 모드 보존과 같은 고급 사용 사례에 대해서는 `kubectl exec` 를 고려해 볼 수 있다.
```

File: content/ko/docs/reference/glossary/pod-disruption.md [hunk 5]
```diff
@@ -14,6 +14,11 @@ tags:
...
-[파드 중단](/ko/docs/concepts/workloads/pods/disruptions/)은 노드에 있는 파드가 자발적 또는 비자발적으로 종료되는 절차이다.
+[파드 중단](/ko/docs/concepts/workloads/pods/disruptions/)은 
+노드에 있는 파드가 자발적 또는 비자발적으로 종료되는 절차이다.
 
-자발적 중단은 애플리케이션 소유자 또는 클러스터 관리자가 의도적으로 시작한다. 비자발적 중단은 의도하지 않은 것이며, 노드의 리소스 부족과 같은 피할 수 없는 문제 또는 우발적인 삭제로 인해 트리거될 수 있다.
+<!--more--> 
+
+자발적 중단은 애플리케이션 소유자 또는 클러스터 관리자가 의도적으로 시작한다. 
+비자발적 중단은 의도하지 않은 것이며, 
+노드의 리소스 부족과 같은 피할 수 없는 문제 또는 우발적인 삭제로 인해 트리거될 수 있다.
```
Comment (yoonian) [inline content/ko/docs/reference/glossary/pod-disruption.md:24]:
```suggestion
노드의 리소스 부족과 같은 피할 수 없는 문제 또는 우발적인 삭제로 인해 트리거가 될 수 있다.
```
헷갈려서 찾아보았습니다. 참고 [온라인가나다](https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=231158)

**Review by yoonian — COMMENTED** [review]
@jihoon-seo 확인해 보았습니다.

**Comment by jihoon-seo** [comment]
@yoonian 리뷰 감사합니다. 리뷰 의견을 모두 반영하였습니다. 😊

**Pull request merged**